cmd/snapshotter/main.go

```diff
@@ -25,7 +26,17 @@ func main() {
 	)
 	flag.Parse()
 
-	sess := session.New()
+	creds := credentials.NewEnvCredentials()
```

Comment:
I think we should not mix `env` and `flags`. Let's add either flags for the AWS credentials or env variables.

Reply:
Don't have a strong opinion about that. So if you think we should not mix it I'm fine with updating this.

Reply:
Then I'd prefer flags 👍